### ISSUE TITLE
Hip Centering in data preprocessing in simple baseline3d

### ISSUE THREAD

**Issue by YuktiADY** [open-mmlab/mmpose#1986]
Hallo,

I have been using this repo for estimating 3d skeleton. Below is my question that i would appreciate if you can answer.

    In the code this is this line which is doing hip centering.
    poses = poses - np.tile( poses[:,:3], [1, len(H36M_NAMES)] )
    poses_set[k] = poses
    Is this necessary to do hip centering on 3d skeleton for estimating 3d pose? Is the hip centering only on 3d skeleton not on 2d ?
    Because i tried without hip centering it seems validation is increased to 5.2

Because i have been training this model on my custom dataset but my validation loss which is 0.45 is not decreasing even after the so much hyperparameter tuning.
2. Also can you please tell me what is correlation between 2d and 3d skeleton. What exactly is done with 3d skeleton.
3. Can you give your suggestion on what can i do to decrease my validation loss(0.45) and training loss(0.112)

Please give your valuable suggestion.

**Comment by ly015** [comment]
Hi,

It is a common practice to predict 3D pose w.r.t. a root joint, due to the ambiguity to predict the absolute global position from 2D images. Some approaches, like [videopose3d](https://arxiv.org/abs/1811.11742), use two different models to predict global root position and relative 3D pose respectively.

3D pose prediction from mono inputs is ill-posed. Pose lifting algorithms like baseline3d build a mapping from the 2D pose to the corresponding 3D pose in a data-driven manner, where the autonomy and kinematics prior as well as the camera parameters are expected to be learned implicitly from the data to alleviate the ill-posedness.

I am not sure why the model didn't converge on your dataset. Maybe you can double-check if the data is well-prepared in the expected format and if the camera parameters are properly set in the data pipeline. You can also start debugging from a mini set of your training data (like one that only has a single sample) and test if the model can overfit it.

**Comment by YuktiADY** [comment]
the validation loss of my model on custom dataset seems to be constant and flat graph. I tried different parameters but it doesnot change.

So the hip centering is done only on 3d skeleton and not on 3d skeleton ?
When i applied without hip centering the loss increases. Do you have an idea ?
Does batch normalization help in improving the loss ?

**Comment by ly015** [comment]
The hip centering can also be applied on 2D skeletons (like in [SimpleBaseline3D](https://github.com/open-mmlab/mmpose/blob/master/configs/body/3d_kpt_sview_rgb_img/pose_lift/h36m/simplebaseline3d_h36m.py#L136-L139)). Generally speaking applying root-centering simplifies the task so the model is more like to converge to higher accuracy.

Batch normalization is already used in the backbone of SimpleBaseline3D by default (see [here](https://github.com/open-mmlab/mmpose/blob/master/mmpose/models/backbones/tcn.py#L189)).
